### ISSUE TITLE
can't import

### ISSUE THREAD

**Issue by AndroidDeveloperLB** [TonnyL/Spark#1]
I get this:

![image](https://user-images.githubusercontent.com/5357526/38175995-1b3c58d2-35ef-11e8-96e1-d65eb380a010.png)

**Comment by TonnyL** [comment]
Hi, @AndroidDeveloperLB 

refer this issue: https://github.com/TonnyL/Charles/issues/2

If you still want to use this library, you could add code below to your **root** `build.gradle` file:

```gradle
allprojects {
    repositories {
        google()
        jcenter()
        maven {
            url "https://dl.bintray.com/tonnyl/maven"
        }
    }
}
```

And if it still not work, try another way:

```gradle
allprojects {
    repositories {
        google()
        jcenter()
        maven {
            url 'https://jitpack.io'
        }
    }
}
```

**Comment by AndroidDeveloperLB** [comment]
@TonnyL I'm not even talking about my project. All I wanted is to try this repository. Cloning it and trying to build it gets this issue.
Even trying to add those to the cloned project doesn't help.
Please fix it, and let us build it without special steps, just like other repositories.

**Comment by TonnyL** [comment]
Hi, @AndroidDeveloperLB 

Have you updated the gradle of Spark? 

If the answer is positive, then I suggest you downgrade it because there is bug of [bintray-release](https://github.com/novoda/bintray-release), which is used in Spark. Refer: https://github.com/novoda/bintray-release/issues/177

**Comment by AndroidDeveloperLB** [comment]
@TonnyL It doesn't matter. Still have the same issue:

![image](https://user-images.githubusercontent.com/5357526/38185501-076afa58-3657-11e8-8ff6-142155980b35.png)

Please just make it work out of the box.

**Comment by TonnyL** [comment]
@AndroidDeveloperLB 

Your gradle version is too high.

<img width="638" alt="screen shot 2018-04-02 at 14 31 24" src="https://user-images.githubusercontent.com/13329148/38185836-94deb818-3682-11e8-81c6-1c626a2add40.png">

When you meet this dialog, DO NOT choose update.

**Comment by TonnyL** [comment]
@AndroidDeveloperLB 

When you meet this dialog

<img width="638" alt="screen shot 2018-04-02 at 14 31 24" src="https://user-images.githubusercontent.com/13329148/38186049-956c4cd6-3683-11e8-952d-2d8c3d2d9e3e.png">

Choose first option instead of third `Update`, there is a bug of  bintray-release if you update gradle to 4.5 or higher.

**Comment by AndroidDeveloperLB** [comment]
@TonnyL I didn't change it for this project. Look at the screenshot. Only thing I've added is:

        maven {
            url "https://dl.bintray.com/tonnyl/maven"
        }
        maven {
            url 'https://jitpack.io'
        }

The current project of the repository uses the latest stable version (3.0.1) .

**Comment by TonnyL** [comment]
@AndroidDeveloperLB  

Remove these code:

```gradle
 maven {
        url "https://dl.bintray.com/tonnyl/maven"
    }
```

It's my fault that I did not notice you are trying to build this lib. I thought you are going to add the lib as dependency😂. 

If you still get a problem, I can help via remote control.

**Comment by AndroidDeveloperLB** [comment]
It wasn't the problem. The gradle was updated before, and when I've reverted the file, it didn't update the folders. Had to remove the gradle folders and now it builds fine.
Because of this, I've decided to report of this issue here:
https://issuetracker.google.com/issues/77335151

Now that it works fine on my device, I have to say that it has a different issue:
I can see the transition has lines. It's a bit hard to explain, but I think anti-alias could help?
